lib/xenops_utils.ml

```diff
@@ -479,16 +479,9 @@ let remap_vgpu vgpu_pci_map vgpu =
       { vgpu with Vgpu.virtual_pci_address = Some addr  } in
   vgpu
 
-let strip x =
-  if x.[String.length x - 1] = '\n'
-  then String.sub x 0 (String.length x - 1)
-  else x
 let get_network_backend () =
   try
-    Unixext.string_of_file !Resources.network_conf
-    |>  strip
-    |>  Stdext.Xstringext.String.split ' '
-    |>  List.hd
+    String.trim (Unixext.string_of_file !Resources.network_conf)
```

Comment:
The new function `strip` is not needed at all, all the files processed by it contain a single line, see https://github.com/xapi-project/xcp-networkd/blob/e231cf37778945b46fb900f11e808e6d86928e12/networkd/network_server.ml#L541